deepsensor/data/loader.py

```diff
@@ -824,7 +810,7 @@ def time_slice_variable(self, var, date, delta_t=0):
             raise ValueError(f"Unknown variable type {type(var)}")
         return var
 
-    def task_generation(
+    def task_generation(  # noqa: D102
```

Comment:
Out of curiosity, what is this `noqa: D102`? Is this comment flag handled automatically by ruff?